content/components/data-table.mdx

```diff
@@ -0,0 +1,604 @@
+/>
+
+</Box>
+</Box>
+
+### Row actions cell options
+
+<!-- TODO: apply a layout -->
+
+<Box display="flex" flexDirection="column" sx={{gap: 3, 'h4': { margin: 0 }}}>
+<Box display="flex" flexDirection={['column', 'column', 'column', 'column', 'row']} sx={{gap: 3}}>
+
+<Box flex={1}>
+
+#### 1 action
+
+Row actions are placed in the last column, and they don't require a visible column header
+
+</Box>
+
+<img
+  width="456"
+  role="presentation"
+  src="https://user-images.githubusercontent.com/2313998/205361234-183d525d-d23d-4e6a-85d7-b0a816c5215f.png"
+/>
+
+</Box>
+
+<Box display="flex" flexDirection={['column', 'column', 'column', 'column', 'row']} sx={{gap: 3}}>
+
+<Box flex={1}>
+
+#### Multiple actions
+
+If you have multiple actions for a row, start by pulling them into a dropdown menu.
+
+</Box>
+
+<img
+  width="456"
+  role="presentation"
+  src="https://user-images.githubusercontent.com/2313998/205361236-f200108e-6fa2-4c38-b0eb-fa2f695c5e15.png"
+/>
+
+</Box>
+
+<Box display="flex" flexDirection={['column', 'column', 'column', 'column', 'row']} sx={{gap: 3}}>
+
+<Box flex={1}>
+
+#### 1 primary action, other actions in an overflow
+
+If one of the actions is heavily used, pull it out for easier access. Do not pull out more than 1 action.
+
+</Box>
+
+<img
+  width="456"
+  role="presentation"
+  src="https://user-images.githubusercontent.com/2313998/205361238-f5219090-dd2f-4c6c-a9ab-6bab8b43d74c.png"
+/>
+
+</Box>
+</Box>
+
+## Interactions
+
+### Sorting
+
+<Box display="flex" flexDirection={['column', 'column', 'column', 'column', 'row']} sx={{gap: 3}}>
+
+<Box as="p" flex={1}>
+  The column header starts unsorted. The first click sorts in ascending order, the second click sorts in descending
+  order, and the third click removes the sort. When the sort is removed, the rows should be in the order they were in
+  when the table was first rendered
+</Box>
+
+<CustomVideoPlayer
+  width="456"
+  loop
+  src="https://user-images.githubusercontent.com/2313998/205755293-c03ee5f6-faa8-4f73-89e5-4bf33cc06086.mp4"
+/>
+</Box>
+
+#### States
+
+<Box display="flex" flexDirection={['column', 'column', 'column', 'column', 'row']} sx={{gap: 3}}>
+
+<Box as="ul" flex={1}>
+  <li>Unsorted (default)</li>
+  <li>Ascending sort active</li>
+  <li>Descending sort active</li>
+  <li>Sort in progress</li>
+</Box>
+
+<img
+  width="456"
+  role="presentation"
+  src="https://user-images.githubusercontent.com/2313998/205361240-85add374-1599-48bf-b6f0-c8e958fbd2dc.png"
+/>
+</Box>
+
+### Filtering
+
+<Box display="flex" flexDirection={['column', 'column', 'column', 'column', 'row']} sx={{gap: 3}}>
+
+<Box as="p" flex={1}>
+  The "query builder" pattern is used to show rows that match the user's query.
+</Box>
+
+<img
+  width="456"
+  role="presentation"
+  src="https://user-images.githubusercontent.com/2313998/205361241-318146a6-d2ff-47e7-96fc-4f1bd4cd5698.png"
```

Comment:
I wasn't sure what the query builder pattern that is described looks like in this image. Based on the anatomy image, is this intended to show the filter present in the table header or is there something else that I should be noting in this image? 👀

Reply:
Oops - I somehow accidentally deleted the filter input 🙈

I'll add it back in

Reply:
I also added a link to the documentation that describes filter inputs